studies/forms.py

```diff
@@ -322,6 +327,7 @@ class Meta:
                 For more information on experiment runner types, please
                 <a href={STUDY_TYPE_HELP_LINK}>see the documentation.</a></p>""",
             "structure": PROTOCOL_HELP_TEXT_INITIAL,
+            "priority": f"This affects how studies are ordered at your lab's custom URL, not the main study page. If you leave all studies at priority 1, all of your lab's active/discoverable studies will be shown in a randomized order on your lab page. If you set the priority of this study to 100, then it will appear last in the list on your lab page. You can find your lab's custom URL from the <a href='/exp/labs/'>labs page</a>. For more info, see the documentation on study prioritization (link here).",
```

Comment:
There are some discrepancies between the text here and the UI. I'm not sure which is correct but we should fix one or the other.
- The text here says "If you leave all studies at priority 1..." but it looks like the default value is 99.
- The text here says "If you set the priority of this study to 100..." but it looks like the max value is 99.

Reply:
Yup, your correct.  I had made a note to myself in the description, but it didn't include the default value.  I'll update both those values.